### ISSUE TITLE
State of python

### ISSUE THREAD

**Issue by JulianGoeltz** [dante-ev/docker-texlive#37]
The CHANGELOG.md states for version 2021-A
> Remove Python, because it is not required in a default setup

however python is advertised both here in the Readme, and on [the docker page](https://hub.docker.com/r/danteev/texlive/). Is is also available locally in the container, but `pip` (or `python -m pip`) fails even though listed on the docker page. Is this intended behaviour, and if so could you please direct people in the readme on how to make `pip` available?
I use this container for a build process that also creates figures, and python including pip is necessary. Tonight the build process failed with `No module named pip`. It would be great to get it to work again, the container is of great help!
Cheers

**Comment by JulianGoeltz** [comment]
Ah, sorry, now I've seen d6569dceb43fdfea8bc411cd692fe15bcae78640 . It seems that the `python` availability is by accident. I fixed it in my workflow simply with an additional
```
apt-get update
apt-get install -y python3-pip
```

I still thing (especially if `python` is installed anyway) that `pip` availability is desirable, but because it is available fixed I'm gonna close the issue. Sorry for the noise

**Comment by wallento** [comment]
I want to second that. It still has Python and adding pip seems a minimal extra, right? It broke my CI by surprise and doc should be updated to be consistent.

**Comment by koppor** [comment]
I was not aware that my "quick fix" broke some other builds. I focused on getting `minted` running. --> https://github.com/dante-ev/docker-texlive/blob/main/test/minimal-minted.tex produces a highlighted pdf.

Note that python comes with the "base" image: https://gitlab.com/islandoftex/images/texlive/-/blob/master/Dockerfile.latest

Thus, "just" pip is missing. - I installed the pygments using debian's packaging system; and not using the "hacky" pip way.

Two users justify for me to include pip in this image.

Will update the CHANGELOG.md, since this is wrong.

Feel free to add PRs on this repo to keep things going. I try to look into it regularly.
